docs/src/content/docs/it/guides/css-and-tailwind.mdx

```diff
@@ -87,112 +91,116 @@ yarn create astro --template starlight/tailwind
 
-   <Tabs>
+    <Tabs>
 
-   <TabItem label="npm">
+    <TabItem label="npm">
 
-   ```sh
-   npm install @astrojs/starlight-tailwind
-   ```
+    ```sh
+    npm install @astrojs/starlight-tailwind
+    ```
 
-   </TabItem>
+    </TabItem>
 
-   <TabItem label="pnpm">
+    <TabItem label="pnpm">
 
-   ```sh
-   pnpm add @astrojs/starlight-tailwind
-   ```
+    ```sh
+    pnpm add @astrojs/starlight-tailwind
+    ```
 
-   </TabItem>
+    </TabItem>
 
-   <TabItem label="Yarn">
+    <TabItem label="Yarn">
 
-   ```sh
-   yarn add @astrojs/starlight-tailwind
-   ```
+    ```sh
+    yarn add @astrojs/starlight-tailwind
+    ```
 
-   </TabItem>
+    </TabItem>
 
-   </Tabs>
+    </Tabs>
 
 3. Crea un file CSS per gli stili di base di Tailwind, ad esempio in `src/tailwind.css`:
 
-   ```css
-   /* src/tailwind.css */
-   @tailwind base;
-   @tailwind components;
-   @tailwind utilities;
-   ```
+    ```css
+    /* src/tailwind.css */
+    @tailwind base;
+    @tailwind components;
+    @tailwind utilities;
+    ```
 
 4. Aggiorna il file di configurazione Astro per utilizzare gli stili di base Tailwind e disabilitare gli stili di base predefiniti:
 
-   ```js {11-12,16-17}
-   // astro.config.mjs
-   import { defineConfig } from 'astro/config';
-   import starlight from '@astrojs/starlight';
-   import tailwind from '@astrojs/tailwind';
-
-   export default defineConfig({
-   	integrations: [
-   		starlight({
-   			title: 'Docs con Tailwind',
-   			customCss: [
-   				// Percorso per gli stili di base di Tailwind
-   				'./src/tailwind.css',
-   			],
-   		}),
-   		tailwind({
-   			// Disattiva gli stili di base predefiniti
-   			applyBaseStyles: false,
-   		}),
-   	],
-   });
-   ```
+    ```js {11-12,16-17}
+    // astro.config.mjs
+    import { defineConfig } from 'astro/config';
+    import starlight from '@astrojs/starlight';
+    import tailwind from '@astrojs/tailwind';
+
+    export default defineConfig({
+    	integrations: [
+    		starlight({
+    			title: 'Docs con Tailwind',
+    			customCss: [
+    				// Percorso per gli stili di base di Tailwind
+    				'./src/tailwind.css',
+    			],
+    		}),
+    		tailwind({
+    			// Disattiva gli stili di base predefiniti
+    			applyBaseStyles: false,
+    		}),
+    	],
+    });
+    ```
 
 5. Aggiungi il plugin Starlight Tailwind a `tailwind.config.cjs`:
 
-   ```js ins={2,7}
-   // tailwind.config.mjs
-   import starlightPlugin from '@astrojs/starlight-tailwind';
+    ```js ins={2,7}
+    // tailwind.config.mjs
+    import starlightPlugin from '@astrojs/starlight-tailwind';
 
-   /** @type {import('tailwindcss').Config} */
-   export default {
-   	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
-   	plugins: [starlightPlugin()],
-   };
-   ```
+    /** @type {import('tailwindcss').Config} */
+    export default {
+    	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+    	plugins: [starlightPlugin()],
+    };
+    ```
```

Comment:
Are you running a specific formatter or a different one than the one configured in the repo? All the changes to these lines will be reverted when Prettier will run (this can be checked using the `pnpm format` command).

Not that this would break anything in this specific case but this makes the diff larger and less readable during the review so I thought I would ask.

Reply:
mhh no, this is very strange. my current workflow is:

1. i sync the files from main
2. i check the i18n tracking site to see what changed and how
3. do a diff between the "en" file and the "it" file
4. update the "it" file

should i resolve this in a commit?

Reply:
Weird, I would guess your editor apply that at the end of the process. Maybe it would be worth checking if it's not configured to use another formatter or has a settings to ignore the project config or something like that. 
In this case, no need to revert as I checked locally and running `pnpm format` will revert the entire changes and not break anything so we are good to go.